backend/corpora/common/entities/dataset.py

```diff
@@ -151,13 +135,11 @@ def create_revision(self) -> "Dataset":
             self.session.add(clone(artifact, dataset_id=revision_dataset.id))
         if self.processing_status:
             self.session.add(clone(self.processing_status, dataset_id=revision_dataset.id))
-        for deployment in self.deployment_directories:
-            self.session.add(clone(deployment, dataset_id=revision_dataset.id))
         self.session.commit()
         return Dataset(revision_dataset)
 
     def tombstone_dataset_and_delete_child_objects(self):
-        self.update(tombstone=True, deployment_directories=[], artifacts=[], processing_status={})
+        self.update(tombstone=True, artifacts=[], processing_status={})
```

Comment:
Same comment as below.. would you need to add the fields `explorer_s3_uri` and `explorer_url` here to update them to be null or the link to indicate that it no longer exists?

Reply:
Some combination of these fields needs to remain. When a user visits the tombstones dataset in explorer, we will need to find it in the database and display the right information about being deleted.

Reply:
Got it, thanks!